src/main/java/com/orange/main/products/service/impl/ProductsServiceImpl.java

```diff
@@ -26,26 +27,44 @@ public class ProductsServiceImpl implements ProductsService {
 
     @Override
     public Products getProductById(Long productId) {
-        Optional<Products> optionalUser =  productsRepository.findById(productId);
-        return optionalUser.get();
+        Optional<Products> product =  productsRepository.findById(productId);
```

Comment:
style: redundant space after equal.
```suggestion
        Optional<Products> product = productsRepository.findById(productId);
```